modules/activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/behavior/IntermediateThrowSignalEventActivityBehavior.java

```diff
@@ -62,5 +68,10 @@ public void execute(ActivityExecution execution) throws Exception {
       leave(execution);
     }
   }
- 
+
+  @Override

```

Comment:
@igdianov 
I'm sorry for pointing at a minor error, but what is the purpose of this override?

Reply:
Removed